.gitattributes

```diff
@@ -0,0 +1,4 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text eol=lf
```

Comment:
I think `*.sh eol=lf` is better because only shell scripts matter. What do you think about it?